sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.controller.service.ts

```diff
@@ -40,4 +50,37 @@ export class ScholasticStandingControllerService {
       ...transformToActiveApplicationDataAPIOutDTO(application, offering),
     };
   }
+
+  /**
+   * Verify if all text file validations were performed with success and throw
+   * a BadRequestException in case of some failure.
+   * @param textValidations validations to be verified.
+   */
+  assertTextValidationsAreValid(
```

Comment:
Why is this method written in controller service? is there an intention to share it between controllers of different client types?

Reply:
No that was not the intention, it was written as an utility method, again following the structure of the education program offering upload.